docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -247,12 +247,7 @@ So it is important to check that the code in the release branch is working befor
    If there are no problems spotted, comment on the Pull Request that the release engineer can continue by following :ref:`the Release section <release>` (do not merge the pull request).
    Otherwise, add comments to the Pull Request for any problems, and comment that they must be resolved before repeating this review process.
 
-#. Merge the release pull request.
-   Do not delete the release branch because it may be used as a base branch for future releases.
-
-#.  Reject the JIRA issue.
-
-    This is necessary because the release process is not "Done" even though no further changes will be made on the branch.
+#.  Accept the JIRA issue, and add a comment that the release process can continue.
```

Comment:
This now sounds like the JIRA issue should be accepted even if there are problems.

Perhaps break the Github step into two substeps: "If there are no problems..." and "Otherwise..." with the correct action on the JIRA issue as a final sentence in each case.

Does this also affect point 1, with the merging into master?